cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
```

Comment:
At a very beginning of the code, it should have the necessary modules.

Required Modules
`import Foundation` 
`import UIKit`

There shouldn't be any unnecessary modules to be imported only the required one should be used.